File: basement/7_function/answer/3.c
#include <stdio.h>
int goukei(int n) {
    int i;
    int sum = 0;
    for (i=1; i<=n; i++) {
        sum += i;
    }
    return sum;
}

int main(void){
    int input;
    scanf("%d", &input);
    printf("%d\n", goukei(input));
}

File: applied/19_bug2/answer/_3.java
import java.util.*;

public class _3 {
    public static void main(String[] args) throws Exception {
        Scanner scan = new Scanner(System.in);
        int input = 0;

        while(input < 1) {
            try {
                input = scan.nextInt();
            } catch (Exception e) {
                input = 0;
                scan.next();
            }
            if (input < 1) {
                System.out.printf("1以上の整数を入力してください\n");
            }
        }
        scan.close();

        int sum = 1;
        for (int i=2; i<=input; i++) {
            sum *= i;
        }
        System.out.printf("%d\n", sum);
    }
}
